# msg/BitrapData.msg
int32 person_id
float64[40] position_vel 
